### DISCUSSION TITLE
Sporadisch, alle Homematic Geräte “Gerätekommunikation gestört”

### DISCUSSION THREAD

**Discussion by cbaum0407** [OpenCCU/OpenCCU#2789] [🇩🇪 Fragen&Antworten]
Hallo

Vorab ein kurzes Info zu meinem System:
HassOS auf einem IntelNUC, RaspberryMatic als AddOn installiert ( 3.75.7.20240601) , die Kommunikation über einen Homematic IP RF USB Stick mit 2m USB-Verlängerung.
Mein Problem:
Sporadisch, nach ca. 1-4 Tagen Betrieb ist kein Homematic Gerät mehr erreichbar. Für alle Geräte wird in der RaspberryMatic “Gerätekommunikation gestört” gemeldet.
Die Geräte sind weder in HA noch in RaspberryMatic bedienbar.

Meine “Lösung”:

-     Neustart der RaspberryMatic über die RaspberryMatic-Systemsteuerung.
-     Ihm HA-Protokoll steht dann:

![2024_06_18 Homematic nicht erreichbar Protokoll](https://github.com/jens-maus/RaspberryMatic/assets/67388458/fc6e73a6-5e73-496d-9f04-976953ea3164)

- Unter HA-Reparaturen:

![2024_06_18 Homematic nicht erreichbar](https://github.com/jens-maus/RaspberryMatic/assets/67388458/5b73f749-36a0-4e0a-8f20-99914b764545)

- Anschliessend die Homematic(IP)lokal Integration neu laden
- 
Jetzt funktioniert wieder alles.
Das ist natürlich keine dauerhafte Lösung.
Ich habe allerdings keinen Plan, wie ich herausfinden kann wodurch dieses Verhalten erzeugt wird. Mir ist auch unklar ob das Problem bei der RaspberyMatic oder bei der Homematic(IP) Integration zu suchen ist. Oder ob es vor der Tastatur sitzt.
Vielleicht hat jemand einen Tipp wie ich vorgehen kann.
Noch was, CarrierSense liegt zwischen 0% und 3% der DutyCycle zwischen 8% und 15%, egal ob der Fehler auftritt oder nicht.

Gruß Christian

**Comment by cbaum0407** [comment]
Hat niemand einen Tip für mich. Mir geht es erstmal nicht um eine Lösung. Ich möchte nur wissen wie gehe ich am besten vor um die Ursache des Verhaltens zu finden.

**Reply by mathiasth** [reply]
Hi, ich habe zwar leider keine Lösung, aber immerhin seit des letzten Updates von Raspberrymatic und HA vor etwa 3 bis 4 Wochen dasselbe wiederkehrende Verhalten, allerdings auf einem RPI 5 mit dem HM-MOD-RPI-PCB, HA und Raspberrymatic laufen in Docker-Containern. Das Setup lief davor tadellos stabil.

**Reply by cbaum0407** [reply]
Da das Problem permanent auftrat habe ich alles umgestellt. Proxmox auf einen MiniPC , HA und RaspberryMatic als VM installiert. Seit dem läufts ohne Probleme.
